Tests/XCDiffCommandTests/CommandsRunnerTests.swift

```diff
@@ -111,6 +111,7 @@ final class CommandsRunnerTests: XCTestCase {
         ✅ LINKED_DEPENDENCIES > "ProjectFramework" target
         ✅ LINKED_DEPENDENCIES > "ProjectTests" target
         ✅ LINKED_DEPENDENCIES > "ProjectUITests" target
+        ✅ ATTRIBUTES > Project Attributes
```

Comment:
Perhaps we can rename "Project Attributes" to "Root project" to be consistent with CONFIGURATIONS and SOURCE_TREES.